examples/LanguageModel/sketch.js

```diff
@@ -0,0 +1,61 @@
+let lm;
+let curWord = 0;
+let angle = 0;
+let radius = 60;
+
+function setup() {
+  createCanvas(400, 400);
+  background(0);
+
+  lm = ml5.languageModel(onModelLoaded);
+}
+
+function draw() {
+  fill(0, 5);
+  rect(0, 0, width, height);
+}
+
+function onModelLoaded() {
+  console.log('Model loaded');
+  select('#generate').removeAttribute('disabled');
+  select('#generate').mouseClicked(generateText);
+}
+
+function generateText() {
+  let prompt = select('#prompt').value();
+  console.log('Prompt: ' + prompt);
+
+  let options = {
+    temperature: 0.9
+  };
+  lm.generate(prompt, options, onToken);
```

Comment:
I love the idea of an `onToken()` event, but for beginners it would be incredibly useful to have an API that mirrors the HuggingFace inference API. In this case, you pass a prompt and a desired number of tokens and you get the entire result back. Something along the lines of:

```javascript
  let options = {
    temperature: 0.9,
    maxTokens: 100
  };
  lm.generate(prompt, options, gotText);
```

I might also consider just be the form `prompt, maxTokens, callback` and then a JS object with more properties if you wanted to set temperature and other more, maybe:

```javascript
  lm.generate(prompt, 100, gotText);
```

and then:

```javascript
  let options = {
    prompt: "How are you?",
    temperature: 0.9,
    maxTokens: 100
  };
  lm.generate(options, gotText);
```

Though maybe it's good to always have the prompt separatel